processor/trilep_analysis.py

```diff
@@ -415,15 +421,22 @@ def fill_multiple_np(hist, arrays, add_sel=dummy, other=None, weight_multiplier=
                     add_sel = sig_sel
                 )
 
-                fill_multiple_np(
-                    output['signal_region_topW'],
-                    {
-                        'lt': lt,
-                        'N': N_SFOS,
-                        'charge': trilep_q,
-                        },
-                    add_sel = sig_sel
-                )
+                for p in self.points:
+                    x,y = p['point']
+                    point = p['point']
+
+                    eft_weight = np.array([self.hyperpoly.eval(self.coeff, point)]*len(ev))
```

Comment:
Not sure if this is correct, self.coeff is the same for every event here? You should rather use the coefficients contained in the event (`ev.Pol`) like here https://github.com/danbarto/tW_scattering/blob/master/processor/SS_analysis.py#L662

Reply:
Done! I remember now why I didn't use ev.Pol... but now I'm not sure how it's being done in SS_analysis. I had to transpose ev.Pol to make the array shapes work in trilep_analysis(eft_weights should be an array with length len(ev), it seems like). Should we be doing the same in SS_analysis?